### DISCUSSION TITLE
Welcome to bxjs-weekly Discussions!

### DISCUSSION THREAD

**Discussion by yamalight** [BuildingXwithJS/bxjs-weekly#120] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swyxio** [comment]
hi! i watch this repo every week because of your great summaries and updates. started recommending to my friends too, its a great way to get updates without clogging up my email inbox!

**Comment by yamalight** [comment]
@sw-yx thanks for the kind words and your continued support! ❤️
